sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -889,6 +978,11 @@ case class Decode(bin: Expression, charset: Expression)
  * (one of 'US-ASCII', 'ISO-8859-1', 'UTF-8', 'UTF-16BE', 'UTF-16LE', 'UTF-16').
  * If either argument is null, the result will also be null.
 */
+@ExpressionDescription(
+  usage = "_FUNC_(string, charset) - Computes the first argument into a BINARY using " +
+    "the provided charset (one of 'US-ASCII', 'ISO-8859-1', 'UTF-8', 'UTF-16BE', " +
+    "'UTF-16LE', 'UTF-16'). If either argument is null, the result will also be null.",
+  extended = "> SELECT _FUNC_('SparkSql', 'UTF-8');\n [53 70 61 72 6B 53 71 6C]")
```

Comment:
On my Mac, the query `select Encode('SparkSql', 'UTF-8');` returns the `SparkSql` in the spark-sql shell.